src/app/app.component.scss

```diff
@@ -20,3 +20,11 @@
     height: 100%;
   }
 }
+#sidebar {
+  width: 500px;
+  min-width: 500px;
+}
```

Comment:
I can't see where the code has been added to determine when the split screen would be shown. I was expecting sidebarRouter.isActivated to be set somewhere I assume that does not already exist?
My understanding is that when active the sidebar is the left pane containing and would contain 'app'. If I have understood correctly then this 500px should be a variable that can be set dynamically from the right pane. And there might be value in centering it and not having 100% width below but that would be for a later discussion.

Reply:
I've added a comment to clarify how/when split screen is shown.

Yes you are correct that the left pane displays the app, not to be confused with the left menu which sits further outside. I.e. currently the building blocks are:

[left menu][content]

Where the [content] further subdivides to
[content-main router][content-side router]

The content-main-router will automatically expand to fit whatever space is available, and the content-side-router requires a static width (default 300px, I've increased to 500px) - but yes this can also be set programatically/dynamically, and both content can have max-widths imposed alongside things like centering if we wanted to, for example, toggle a display mode that rendered the content at a specific width

Reply:
I think it would be worth adding being able to change both the side-router width and the actual content width within that dynamically to be able to see what the content looks like on various devices without changing screen layout. Currently the RCT device is going to have 480 width so it would be good to have that as a default.

Reply:
Sounds good - I'll link the specific-width changes into the next PR more specifically for the feedback system (drafted in #1603)